packages/application/src/shell.ts

```diff
@@ -302,26 +302,45 @@ export class LabShell extends Widget implements JupyterFrontEnd.IShell {
     spacer.id = 'jp-top-spacer';
     this.add(spacer, 'top', { rank: 1000 });
 
-    const label = document.createElement('label');
-    label.textContent = 'Single-Document Mode';
-    label.title = 'Single-Document Mode';
-    const checkbox = document.createElement('input');
-    checkbox.type = 'checkbox';
-    checkbox.value = 'single-document';
-    checkbox.title = 'Single-Document Mode';
+    // switch accessibility refs:
+    // https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Switch_role
+    // https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#Accessibility_concerns
+    const sdmSwitch = document.createElement('button');
```

Comment:
We might want to start using the same switch components now that there is more than a single instance of a switch in lab.

There was a discussion about this when it was first introduced in the debugger extension here: https://github.com/jupyterlab/debugger/pull/376

Which is now tracked here: https://github.com/jupyterlab/jupyterlab/issues/8010